src/TagField.php

```diff
@@ -446,7 +450,7 @@ public function validate($validator)
      */
     public function performReadonlyTransformation()
     {
-        $copy = $this->castedCopy(ReadonlyTagField::class);
```

Comment:
This is the correct class, please rebase or undo this change

Reply:
Yup thats fixed ;)